core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -1015,22 +1011,22 @@ private object RTS {
   }
 
   sealed trait FiberStatus[E, A] {
-    def error: Option[Throwable]
+    def errors: Option[List[Throwable]]
   }
   object FiberStatus {
-    final case class Executing[E, A](error: Option[Throwable],
+    final case class Executing[E, A](errors: Option[List[Throwable]],
```

Comment:
Looks good: `None` means it's not been killed; `Some` means it has been killed, possibly with some errors.